src/tiled/terraindock.cpp

```diff
@@ -302,6 +314,8 @@ void TerrainDock::setCurrentTerrain(Terrain *terrain)
     mEraseTerrainButton->setChecked(terrain == nullptr);
 
     mRemoveTerrainType->setEnabled(terrain != nullptr);
+    mMoveTerrainTypeUp->setEnabled(terrain != nullptr);
+    mMoveTerrainTypeDown->setEnabled(terrain != nullptr);
```

Comment:
Could you look into also enabling/disabling these actions as appropriate based on the currently selected terrain? So that when the first terrain is selected, the "move terrain up" action is disabled?

Reply:
This has been implemented in e637e0c